### ISSUE TITLE
Bug: Entity xx implements device_state_attributes

### ISSUE THREAD

**Issue by gateriderz** [JurajNyiri/HomeAssistant-Tapo-Control#123]
**Describe the bug**
Warning in Home Assistant 2021.12
```
WARNING (MainThread) [homeassistant.helpers.entity] Entity <> implements device_state_attributes. Please report it to the custom component author.
```

**To Reproduce**
1. Install Home Assistant Core 2021.12
2. Check Logs

**Expected behavior**
No warning errors

**Log**
[homeassistant.helpers.entity] Entity ?? <> implements device_state_attributes. Please report it to the custom component author.

**Camera (please complete the following information):**
All Cameras

**Using stream component**
yes

**Camera works through onvif component OR different cameras work in general**
yes

**Camera has all attributes filled out properly in developer tools**
yes

**HASS Environment**
HASS OS

**Additional context**
Change device_state_attributes --> extra_state_attributes

```
camera.py
@property
    def extra_state_attributes(self):
        return self._attributes
```

**Comment by JurajNyiri** [comment]
Fixed in [3.4.3](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/3.4.3).

**Comment by gateriderz** [comment]
> Fixed in [3.4.3](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/3.4.3).

Thank you @JurajNyiri for the quick resolution.

There is 1 more entry that needs to be retrofited:
```
switch.py
class LocaltuyaSwitch
    @property
    def device_state_attributes(self):
```

Thank you

**Comment by JurajNyiri** [comment]
@gateriderz I am not maintainer of localtuya, and there is already an issue opened there https://github.com/rospogrigio/localtuya/issues/658
